src/sonic-config-engine/minigraph.py

```diff
@@ -1271,6 +1271,13 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         'synchronous_mode': 'enable'
         }
     }
+    results['FLEX_COUNTER_TABLE'] = {
+        'ACL': {
+            'FLEX_COUNTER_STATUS': 'enable',
+            'FLEX_COUNTER_DELAY_STATUS': 'true',
+            'POLL_INTERVAL': '10000'
+        }
+    }
```

Comment:
And can you please add test script to cover this change? Thanks

Reply:
There was no plan to configure FC from minigraph, considering minigraph was deprecated long time ago.
The reason I put it here is I want to make sure when generating config from minigraph FC ACL counters is enabled.
But that actually is already enabled since the init_cfg.json has FC ACL enabled.